### DISCUSSION TITLE
Support transforming to `OpenAPI 3.0` documentation

### DISCUSSION THREAD

**Discussion by shuqingzai** [spatie/laravel-data#490] [Ideas]
At present, I see that [laravel-typescript-transformer](https://github.com/spatie/laravel-typescript-transformer) can support transforming to `TypeScript`. Is there any plan to support transforming to `OpenAPI 3.0`? The process of writing documents is too cumbersome. I hope to support one-click transform and save a lot of time

reference:

- [OpenAPI Specification](https://swagger.io/specification/)
- [OpenAPI 3.0.3](https://spec.openapis.org/oas/v3.0.3#version-3-0-3)

**Comment by rubenvanassche** [comment]
You're always welcome to built a package implementing this, I would happily link to it. At the moment we don't see this as a core feature of the package.
